pheweb-test/README.md

```diff
@@ -0,0 +1,137 @@
+# Pheweb Integration
+
+## What will Pheweb do?
+
+Pheweb shows correlations between genotypes and phenotypes by looking at gene variants
+
+e.g. What is the correclation between the Braca-1 gene and Breast Cancer?
+
+Pheweb also provides the abilito to search by gene, mutation and phenotype to see what correlations are calculated
+
+Pheweb is strictly a viewer/visulizer, it does not perform any analysis.
+
+## Dependency list
+
+These are some of the dependency issues I ran into while trying different things out with PheWeb:
+
+1. pysam
+2. cython
+3. python environment
+4. libbz2-dev or bzip2-devel
+5. pyliblzma
+6. liblzma-dev
+
+All of these dependencies seem to be available on app engine standard.
+
+### Google Cloud Dependencies for Local Development
+
+1. google-cloud-sdk-app-engine-python
+2. cloud-datastore-emultator
+
+It should be noted that PheWeb is finicky about the Python version in use. I have successfully run PheWeb with Python 3.6 and 3.8
+
+## Does it need a live web server?
+
+1. Pheweb can run off of gunicorn
+2. Requires Python 3.8 (or 3.6)
+
+If you want to run pheweb with gunicorn you must first run `PheWeb wsgi`
+
+This will startup a [Flask](https://flask.palletsprojects.com/) server. The server will serve up static HTML files ([example](https://github.com/statgen/pheweb/blob/master/pheweb/serve/templates/gene.html)) that manually pull in all required javascript files using `script` tags.
```

Comment:
Thanks - this is the critical piece I was looking for. It does server-side rendering using Flask templates, i.e. the server is actually necessary. static assets are clearly marked under the static/ folder, which we could push onto a CDN.